.github/workflows/lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Lints
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    # Setup nodejs and build the webui so that we can 
+    # compile the go code that embeds the webui.
+    - name: Setup node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 21
```

Comment:
I think we have a file in the repo that specifies the correct version to use?

Reply:
Yes, .tool-versions says 

```
python 3.10.5
nodejs 21.5.0
```

Reply:
interesting question - should we reuse .tool-version? i suspect yes

Reply:
Yeah, my point is that the action should be reading the file to know the correct version to load.

Reply:
Have added this now, and also added go to the list of languages.  Will have to remember to keep .tool-versions in sync with the go.mod now as well.